### PR TITLE
Promote Oríon's agnosticity

### PR DESCRIPTION
**Pull request by Thomsch**
# Description
Adds a line to promote the ability of Oríon to work with any language and support for any configuration file.

# Checklist
## Tests
- [x] All new and existing tests are passing (`$ tox -e py38`; replace `38` by your Python version if necessary)

## Documentation
- [x] I have updated the relevant documentation related to my changes

## Quality
- [x] I have read the [CONTRIBUTING](https://github.com/Epistimio/orion/blob/develop/CONTRIBUTING.md) doc
- [x] My commits messages follow [this format](https://chris.beams.io/posts/git-commit/)
- [x] My code follows the style guidelines (`$ tox -e lint`)

### CODE REVIEW

**Comment by bouthilx** [comment]
I feel we should give a pointer for this item as well otherwise it is likely difficult to understand what we mean. We don't have any clear documentation for the language agnostic part, probably something we should add, but for the configuration we have this: https://orion.readthedocs.io/en/latest/user/searchspace.html#configuration-file

**Comment by Thomsch** [comment]
I agree! I wanted to put a link but couldn't find any relevant resources. Thank you for the link.
Maybe we could add a paragraph or two in https://orion.readthedocs.io/en/latest/install/gettingstarted.html#integration ?

**Comment by bouthilx** [comment]
> Maybe we could add a paragraph or two in https://orion.readthedocs.io/en/latest/install/gettingstarted.html#integration ?

Good idea, that would be the right place I think! Would you want to do it in this PR or we create a card for later?

File: docs/src/user/script.rst [hunk 1]
```diff
@@ -54,6 +54,11 @@ the local one passed to
 created. To access the particular working directory of a trial, see next sections
 :ref:`commandline_templates` and :ref:`env_vars`.
 
+Language compatibility
+======================
+The command line works for scripts and programs in any language.
+The only requirement is that the executed script returns a JSON string with the objective value.
+
```
Comment (bouthilx) [inline docs/src/user/script.rst:61]: The format is
```
[
    {
        'name': 'some-objective',
        'type': 'objective',
        'value': 1
    }
]
```

See https://orion.readthedocs.io/en/latest/code/client.html#orion.client.report_results for more details.

Comment (Thomsch) [reply docs/src/user/script.rst:61]: Very good!

**Pull request merged**